README.md

```diff
@@ -71,6 +71,7 @@ eth-account | Ethereum account ID for the bot
 eth-password | Ethereum account password for the bot
 eth-rpc-url | RPC URL to Ethereum node, e.g. Geth. Either local or remote
 eth-wallet-file | Location of wallet file. If Geth is run with the parameters as given below, it will reside under `$HOME/.ropsten/keystore`
+offline-signing | Specifies whether to sign transactions locally before sending. Default is false. Set to true when connecting to e.g. Infura or any remote Geth node that does not unlock accounts
```

Comment:
Proposal: default it to true, since that way we fix a security issue that we have right now: having the node unblocked.

Reply:
Agree, changed to true, both there and in `eth/core.clj`.

Reply:
Could you change it in `env/dev/resources/config.edn` as well?

Reply:
Done:)